src/components/PendingSwapExchange.tsx

```diff
@@ -38,20 +38,20 @@ const PendingSwapExchange = ({
   const { t } = useTranslation()
   const { synthTokenFrom, tokenTo, synthBalance, swapType } = pendingSwap
   const formattedSynthBalance = commify(
-    formatBNToString(synthBalance, synthTokenFrom.decimals, 6),
+    formatBNToString(synthBalance, synthTokenFrom?.decimals ?? 0, 6),
```

Comment:
these changes are all moot if you don't add `| undefined` to `PendingSwap` type, see comments in `usePendingSwapData`